src/pages/ProfilePage/Account.js

```diff
@@ -76,10 +77,13 @@ class ProfilePage extends React.Component {
 
 	componentDidMount() {
 		const {
-			usecase, deploymentTimeframe, phone, company, username, picture,
+			usecase, deploymentTimeframe, phone, company, username, picture, isShopify,
 		} = this.props; // prettier-ignore
+
+		const defaultUseCase = isShopify ? useCaseOptions[2] : usecase;
+
 		this.profileForm.patchValue({
-			usecase,
+			usecase: usecase || defaultUseCase,
```

Comment:
@jyash97 I'm not sure this line is needed here.

Reply:
I removed this and checked it didnt work. So I added this again.